core/mixins.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
+from app import db
+
+# Creating the customer model
+class BaseModelMixen(object):
+    id = db.Column(db.Integer, primary_key=True)
+    date_created = db.Column(db.DateTime, default=datetime.utcnow)
+    # Nullable default is true, Unique default is False
+
+    def save(self):
+        # Used to save data
+        db.session.add(self)
+        return db.session.commit()
```

Comment:
one thing I'm going to add as a last minute requirement: this method shouldn't return anything